entity-framework/core/querying/tracking.md

```diff
@@ -2,42 +2,46 @@
 title: Tracking vs. No-Tracking Queries - EF Core
 description: Information on tracking and no-tracking queries in Entity Framework Core
 author: smitpatel
-ms.date: 11/15/2021
+ms.date: 3/15/2023
 uid: core/querying/tracking
 ---
 # Tracking vs. No-Tracking Queries
 
-Tracking behavior controls if Entity Framework Core will keep information about an entity instance in its change tracker. If an entity is tracked, any changes detected in the entity will be persisted to the database during `SaveChanges()`. EF Core will also fix up navigation properties between the entities in a tracking query result and the entities that are in the change tracker.
+Tracking behavior controls if Entity Framework Core keeps information about an entity instance in its change tracker. If an entity is tracked, any changes detected in the entity are persisted to the database during `SaveChanges`. EF Core also fixes up navigation properties between the entities in a tracking query result and the entities that are in the change tracker.
 
 > [!NOTE]
 > [Keyless entity types](xref:core/modeling/keyless-entity-types) are never tracked. Wherever this article mentions entity types, it refers to entity types which have a key defined.
 
-> [!TIP]
-> You can view this article's [sample](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Querying/Tracking) on GitHub.
```

Comment:
Why this change?

Reply:
IMAO, too much bling can be distracting. I can revert.

Reply:
Yes, please.